### PR TITLE
Todo Example - hooks + Relay 11.0.2

### PR DESCRIPTION
**Pull request by levibuzolic**
This is a quick migration of the Todo example to use the `useFragment` and `useLazyLoadQuery` hooks.

### CODE REVIEW

File: todo/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "private": true,
   "scripts": {
     "start": "babel-node ./server.js",
-    "build": "relay-compiler-experimental --src ./js/ --schema ./data/schema.graphql --artifactDirectory ./__generated__/relay",
```
Comment (mrtnzlml) [inline todo/package.json:5]: @levibuzolic Is this change intentional? It was changed to experimental not so long time ago: https://github.com/relayjs/relay-examples/commit/98a5921883ebc38a959a431c70125128f79fdcf3

Comment (levibuzolic) [reply todo/package.json:5]: Ah, I mistakenly thought `relay-compiler@11.0.2` was ahead of `relay-compiler-experimental`. 

Thanks for pointing it out, I'll roll this change back. 🙇‍♂️

Comment (levibuzolic) [reply todo/package.json:5]: Digging into it a bit more, it appears the `relay-compiler-experimental` package is just the Rust binary rewrite of the compiler and is functionally equivalent (at least for the output of the Todo app). 

I'll roll this change back for now but will open a separate PR to target the non-experimental package here as it seems like the more ideal version to target for an OSS example repo while the experimental compiler is still in early development.

**Comment by alunyov** [comment]
@levibuzolic thanks for PR.

We still need to keep this app to use QueryRenderer/Containers APIs - as we're using it for a release sanity test to make sure these (legacy) APIs are still working.

For the hooks, we have `issue-tracker` example that serves the same purpose: basic sanity check before the release.

**Comment by levibuzolic** [comment]
Thanks for the context @alunyov, I’ll close this PR. 👍
